### ISSUE TITLE
Terraform Registry - Support

### ISSUE THREAD

**Issue by mgomezzul** [transcend-io/terragrunt-atlantis-config#172]
Hello! Recently Terragrunt team added the following feature to get terraform modules directly from the Terraform Registry, so with this feature you don't need to put the github repository as source.

I guess terragrunt-atlantis-config can not parse this new syntax

```
terraform {
  source = "tfr:///cloudposse/kms-key/aws?version=0.10.0"
}
```
I'm getting this error:

```
Error: Failed to read module directory: Module directory /home/atlantis/.atlantis/repos/xxx/yyy/zzz/aws/xxx/zzz/kms/sops/tfr:/cloudposse/kms-key/aws?version=0.10.0 does not exist or cannot be read.
```

Let me know  how can I help.

Thanks in advance

**Comment by dmattia** [comment]
I love it, thank you both! @henworth If you pop that into a PR, I'll gladly merge it 😄 

Otherwise I can do so this Friday

**Comment by john-delivuk** [comment]
It appears http module sources are also not supported

**Comment by henworth** [comment]
This is true, the current comparison is very naive to all the Terraform supported source methods. I think a better way might be to check what the source method is and if it matches one of the supported remote methods (maybe via a regex like `/^(http|tfr|git)\:/`) then bypass all the file path resolution logic. What do you think @dmattia?

**Comment by john-delivuk** [comment]
I have a fix incoming. To my understanding only local paths will ever function correctly.

**Comment by dmattia** [comment]
This should be fixed in v1.11.0: https://github.com/transcend-io/terragrunt-atlantis-config/releases/tag/v1.11.0

Please let me know if this doesn't work for you 😄
